docs/release_process_quarterly.md

```diff
@@ -0,0 +1,501 @@
+        - [ ] test SPaT broadcast rate event
+        - [ ] test SPaT minimum data event
+        - [ ] test SPaT time change details event
+        - [ ] test stop line passage events
+        - [ ] test stop line stop events
+
+### 3. Project Reference Updates & Release Creation
+    - [ ] Update version number in pom.xml file for the `jpo-conflictmonitor` project if not already done.
+    - [ ] Update git submodule & artifact references for the ‘jpo-conflictmonitor project.
+        - [ ] Open the jpo-conflictmonitor project in an IDE.
+        - [ ] Navigate to the jpo-utils directory and run `git checkout tags/jpo-utils-x.x.x` to update the submodule reference.
+        - [ ] Update the version number in the pom.xml files for the jpo-geojsonconverter and jpo-ode-* dependencies to match the version number of the corresponding releases. (e.g. 1.0.0)
+            - [ ] Update the jpo-conflictmonitor/pom.xml
+            - [ ] Update the message-sender/pom.xml
+        - [ ] Commit these changes to the `release/(year)-(quarter)` branch & push the changes to the remote repository.
+        - [ ] Ensure these changes pass CI/CD checks before continuing.
+    - [ ] Merge `release/(year)-(quarter)` branch into `master` branch for the jpo-conflictmonitor project, and create a release with the version number of the release. (e.g. jpo-conflictmonitor-x.x.x)
+    - [ ] Create docker image
+        - [ ] Use the release for the jpo-conflictmonitor to produce docker image with the same tag name, containing the version number.
+        - [ ] Upload docker image to [DockerHub](https://hub.docker.com/u/usdotjpoode)
+        - [ ] Tag docker image with the version number of the app. (e.g. 1.0.0)
+        - [ ] Tag docker image with year and quarter of release. (e.g. 2024-Q2)
+        - [ ] Tag docker image with 'latest' tag for the most recent release.
+    - [ ] Merge master branch into develop branch & verify that CI/CD passes.
+
+### 4. DockerHub Image Testing
+    - [ ] image starts up correctly
+    - [ ] program processes SpAT/Map/BSM messages and generates events as expected (see https://github.com/usdot-jpo-ode/jpo-conflictmonitor/wiki/Integration-Tests)
+        - [ ] test BSM events
+        - [ ] test connection of travel event
+        - [ ] test intersection reference alignment events
+        - [ ] test lane direction of travel event
+        - [ ] test Map broadcast rate event
+        - [ ] test Map minimum data event
+        - [ ] test signal group alignment events
+        - [ ] test signal state conflict events
+        - [ ] test SPaT broadcast rate event
+        - [ ] test SPaT minimum data event
+        - [ ] test SPaT time change details event
+        - [ ] test stop line passage events
+        - [ ] test stop line stop events
+
+
+## jpo-conflictvisualizer
+### Prerequisites
+    - [ ] jpo-ode release completed
+    - [ ] jpo-geojsonconverter release completed
+    - [ ] jpo-conflictmonitor release completed
```

Comment:
The visualizer also has a direct dependency on asn1_codec (it includes it as a submodule)

Nature of dependencies:

    - [ ] jpo-ode  (Maven)  releasecompleted
    - [ ] jpo-geojsonconverter (Maven) release completed
    - [ ] jpo-conflictmonitor (Maven) release completed
    - [ ] asn1_codec (submodule)

Reply:
Addressed!